src/components/form/select/index.js

```diff
@@ -9,11 +9,14 @@ const Select = ({ register, options, name, error, placeholder, className = '', .
       <option value="" disabled selected className="placeholder">
         {placeholder}
       </option>
-      {options.map((value, i) => (
-        <option key={i} value={value}>
-          {value.label ? value.label : value}
-        </option>
-      ))}
+      {options.map((value, i) => {
+        console.log(value);
+        return (
+          <option key={i} value={value.topic ? value.topic.id : value}>
+            {value.topic ? value.topic.label : value}
+          </option>
```

Comment:
The `Select` component defined in this file should be generic. It shouldn't check for a `topic` object in the `options` prop. Instead, you should pass a proper `options` prop for the `Select` component at `src/pages/createTarget/index.js`.

All you should do is to pass an array of objects to the `options` prop that contains a `label` property for showing it to the user.